beacon_node/lighthouse_network/src/service/mod.rs

```diff
@@ -553,9 +549,9 @@ impl<E: EthSpec> Network<E> {
             // strip the p2p protocol if it exists
             strip_peer_id(&mut multiaddr);
             match self.swarm.dial(multiaddr.clone()) {
-                Ok(()) => debug!(self.log, "Dialing libp2p peer"; "address" => %multiaddr),
+                Ok(()) => debug!( address = %multiaddr, "Dialing libp2p peer"),
```

Comment:
Lots of these lines have unnecessary spaces which can be removed
```suggestion
                Ok(()) => debug!(address = %multiaddr, "Dialing libp2p peer"),
```